js/navbar.js

```diff
@@ -1,3 +1,17 @@
+window.addEventListener('DOMContentLoaded',(event)=>{
+  const checkURL = window.location.href;
+  const stagingURL = "https://staging-welcome.realdevsquad.com/";
+  const productionURL = "https://welcome.realdevsquad.com/";
+
+  if (checkURL == stagingURL) {
+    document.getElementById("user-login").href = "https://github.com/login/oauth/authorize?client_id=c4a84431feaf604e89d1";
```

Comment:
use a separate const variable for referencing `user-login`.

Reply:
But here, user-login is just the id for anchor tag.
Do i have to this: const login="user-login";
document.getElementById(login).href = "o auth link";